src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -168,14 +175,14 @@ def __init__(self):
             self._jm = JobManager()
         if self._msg_map is None:
             self._msg_map = {
-                MESSAGE_TYPE["CANCEL"]: self._cancel_jobs,
-                MESSAGE_TYPE["CELL_JOB_STATUS"]: self._get_job_states_by_cell_id,
-                MESSAGE_TYPE["INFO"]: self._get_job_info,
-                MESSAGE_TYPE["LOGS"]: self._get_job_logs,
-                MESSAGE_TYPE["RETRY"]: self._retry_jobs,
+                MESSAGE_TYPE["CANCEL"]: self.cancel_jobs,
+                MESSAGE_TYPE["CELL_JOB_STATUS"]: self.get_job_states_by_cell_id,
+                MESSAGE_TYPE["INFO"]: self.get_job_info,
+                MESSAGE_TYPE["LOGS"]: self.get_job_logs,
+                MESSAGE_TYPE["RETRY"]: self.retry_jobs,
                 MESSAGE_TYPE["START_UPDATE"]: self._modify_job_updates,
-                MESSAGE_TYPE["STATUS"]: self._get_job_states,
-                MESSAGE_TYPE["STATUS_ALL"]: self._get_all_job_states,
+                MESSAGE_TYPE["STATUS"]: self.get_job_states,
+                MESSAGE_TYPE["STATUS_ALL"]: self.get_all_job_states,
```

Comment:
renamed these methods for clarity and to make a better distinction between top level and internal methods